pybossa/importers/importer.py

```diff
@@ -238,11 +239,12 @@ def create_tasks(self, task_repo, project, importer=None, **form_data):
         if n==0:
             msg = gettext('It looks like there were no new records to import. ')
         elif n == 1:
-            msg = str(n) + " " + gettext('new task was imported successfully ')
+            msg = str(n) + " " + gettext('new task was imported successfully. ')
         else:
-            msg = str(n) + " " + gettext('new tasks were imported successfully ')
+            msg = str(n) + " " + gettext('new tasks were imported successfully. ')
         msg += str(validator)
-
+        if data_access_levels and 'data_access' in importer.headers():
+            msg += gettext('Task data_access column will not be compared against projects data access.')
```

Comment:
Should we consider re-wording this warning to users, similar to:

"The column "data_access" will be ignored. Please configure task level data access on the Project Settings page."

Reply:
"data_access" column won't be ignored and will be part of task payload as a regular payload. The data access settings on Project settings page are considered as project data access settings. Thus, I think the above message would be good.